### PR TITLE
Update Risk Level and test cases

### PR DESCRIPTION
**Pull request by Sheeepen**

### CODE REVIEW

**Review by SimWPEric — COMMENTED** [review]
Changes from `Tag` to `RiskLevel` looks good. 
From my understanding, multiplicity for RiskLevel is 0..1, meaning a Student may not be tagged to a RiskLevel initially ie: haven't visit the counsellor, after the counselling, student can then be tagged to a RiskLevel. I think there's a need to make changes to files such as `AddCommandParser` to change `Set<RiskLevel> tagList` to simply `RiskLevel`

File: src/test/java/seedu/address/logic/parser/ParserUtilTest.java
Comment (darrentfy) [inline src/test/java/seedu/address/logic/parser/ParserUtilTest.java:1]: Good job babe 😋 
-Kong

**Review by darrentfy — APPROVED** [review]
LGTM, thanks for updating all the test cases related to risk level

File: src/test/java/seedu/address/logic/parser/ParserUtilTest.java
Comment (Sheeepen) [reply src/test/java/seedu/address/logic/parser/ParserUtilTest.java:1]: thank you :godmode:

File: src/main/java/seedu/address/model/risklevel/RiskLevel.java [hunk 1]
```diff
@@ -3,19 +3,18 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
-import java.util.HashSet;
 import java.util.Set;
 
+import seedu.address.model.util.LimitedHashSet;
```
Comment (quzhetao01) [inline src/main/java/seedu/address/model/risklevel/RiskLevel.java:8]: Good use to Limited HashSet

File: src/main/java/seedu/address/model/util/LimitedHashSet.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+public class LimitedHashSet<T> extends HashSet<T> {
+
+    private int maxSize;
+
+    public LimitedHashSet(int maxSize) {
+        this.maxSize = maxSize;
+    }
+
+    @Override
+    public boolean add(T element) {
```
Comment (quzhetao01) [inline src/main/java/seedu/address/model/util/LimitedHashSet.java:19]: perhaps can throw an exception here instead

**Review by quzhetao01 — APPROVED** [review]

**Pull request merged**